facia-tool/public/package.json

```diff
@@ -8,17 +8,17 @@
     "configFile": "js/jspm-config.js",
     "dependencies": {
       "EventEmitter": "npm:wolfy87-eventemitter@4.2.11",
-      "css": "^0.1.0",
-      "font-awesome": "4.2.0",
-      "jquery": "2.1.3",
+      "css": "github:systemjs/plugin-css@^0.1.0",
+      "font-awesome": "npm:font-awesome@4.2.0",
+      "jquery": "github:components/jquery@2.1.3",
```

Comment:
lol just realised it is facia-tool. If github starts behaving, teamcity will build #8731, I'll merge that, you can rebase/merge and it'll go away.